rmw_connext_cpp/include/rmw_connext_cpp/connext_static_publisher_info.hpp

```diff
@@ -54,4 +65,95 @@ class ConnextPublisherListener : public DDS::PublisherListener
   std::atomic<std::size_t> current_count_;
 };
 
+/**
+ * Remap the specific RTI Connext DDS DataWriter Status to a generic RMW status type.
+ *
+ * @param mask input status mask
+ * @param event
+ */
+inline rmw_ret_t ConnextStaticPublisherInfo::get_status(const DDS_StatusMask mask, void * event)
```

Comment:
We should bring this function to a cpp file :)

Reply:
Not sure, but I broke them out into implementation files.